packages/chrony/chronyd.service

```diff
@@ -1,12 +1,12 @@
 [Unit]
 Description=A versatile implementation of the Network Time Protocol
 Documentation=https://chrony.tuxfamily.org
-After=network-online.target configured.target
-Requires=network-online.target configured.target
+After=network-online.target
+Requires=network-online.target preconfigured.target
 
 [Service]
 Type=simple
 ExecStart=/usr/sbin/chronyd -d -F -1
 
 [Install]
-WantedBy=multi-user.target
+WantedBy=preconfigured.target
```

Comment:
I used `WantedBy=preconfigured.target` because I thought that chrony should is one of the services required at the very early stages of the boot (during the preconfigured target). I'll make the changes to be started in the `configured` target.

Reply:
It'd be fine to put it in the "preconfigured" stage, although a bit tricky since chrony itself needs to be configured by settings. I'm OK with "configured" for now.